semver.md

```diff
@@ -7,9 +7,10 @@ Summary
 Given a version number MAJOR.MINOR.PATCH, increment the:
 
 1. MAJOR version when you make incompatible API changes,
-1. MINOR version when you add functionality in a backwards compatible
+1. MINOR version when you add functionality to the API in a backwards compatible
```

Comment:
where but to the API could you possibly add functionality?

Reply:
I don't know, but since this is the terminology used by the specification elsewhere, I thought it was appropriate here as well.

> 1. MAJOR version when you make incompatible API changes,

where but to the API could you possibly make incompatible changes?

I can remove this change if it makes it easier to get this PR merged. The important part is to make clear that you can make patch-level changes which are not bug fixes. The specification's ambiguity regarding this has made people come to the wrong conclusion that changes that are neither changes to the API, nor bug fixes MUST be released in a MINOR version. My proposed wording that you commented on makes this mistake almost impossible so IMHO it's strictly better with it than without.